### ISSUE TITLE
tzfile.lua:199 got Unsupported version error for Asia/Jerusalem

### ISSUE THREAD

**Issue by pahud** [daurnimator/luatz#7]
```
> luatz.get_tz('Asia/Jerusalem')
/usr/local/share/lua/5.1/luatz/tzfile.lua:199: Unsupported version
stack traceback:
	[C]: in function 'error'
	/usr/local/share/lua/5.1/luatz/tzfile.lua:199: in function 'read_tz'
	/usr/local/share/lua/5.1/luatz/tzfile.lua:205: in function 'read_tzfile'
	/usr/local/share/lua/5.1/luatz/tzcache.lua:24: in function 'get_tz'
	stdin:1: in main chunk
	[C]: ?
> 
```

Please help

thanks

**Comment by daurnimator** [comment]
What version of luatz are you using? You can check with: `luarocks show luatz`

My first guess is that you're using an old luatz release and ran into a version 3 tzfile. (see issue #2)

**Comment by daurnimator** [comment]
Ah sorry. It looks like my (partial) fix for issue #2 wasn't enough/correct. Working on it now.

**Comment by daurnimator** [comment]
Please confirm that fix works for you :)
